app/src/lib/git/branch.ts

```diff
@@ -78,7 +78,13 @@ export async function deleteBranch(
     await deleteLocalBranch(repository, branch.name)
```

Comment:
I think the comment/documentation for `deleteBranch` should be updated to reflect the new behavior, right? (Maybe after considering the rest of the feedback in this PR)

(Sorry, I couldn't post this comment in the lines of the function documentation, this is as far as I could go 😆 )